MuhanguziDavid/url-shortener
# Set up basic node project with home and page not found routes

### What does this PR do?
- This PR sets up the node project with handlebars view engine.
- The home route and `not found` route are setup.

### How should I test this PR?
- Clone this Repo: `git clone https://github.com/MuhanguziDavid/url-shortener.git`
- cd into the project directory
- run `npm install` to install project dependencies
- run npm start the app
- open `http://localhost:8080/` in your browser. You should see the home page
- add any path to the URL. You should get feedback that the page was not found
